### DISCUSSION TITLE
Default ltsp.conf file

### DISCUSSION THREAD

**Discussion by renatogfilhinho** [ltsp/ltsp#870] [General] [answered]
A question about the location of the default ltsp.conf file: After installing all the packages and running all the ltsp commands, I see that everything works beautifully, but when accessing the /etc/ltsp folder I see that the ltsp.conf file does not exist. What is the default file that the initial ltsp uses and from the documentation I saw that I can create one with the install command, is this generated file the same as the one that the ltsp service initially uses, by default? Thank you very much !!!

**Comment by alkisg** [comment, marked as answer]
By default, no configuration file exists and no configuration file is used.
To create one, you use the initial command that you saw in the documentation:
https://ltsp.org/man/ltsp.conf

**Reply by renatogfilhinho** [reply]
Thank you very much!
